pkg/network/protocols/http/ebpf_javatls.go

```diff
@@ -48,7 +53,13 @@ func newJavaTLSProgram(c *config.Config) *JavaTLSProgram {
 		log.Errorf("java TLS support requires runtime-compilation to be enabled")
 		return nil
 	}
-	javaUSMAgentJarPath = filepath.Join(c.JavaDir, "agent-usm.jar")
+	javaUSMAgentJarPath = filepath.Join(c.JavaDir, AgentUSMJar)
+	jar, err := os.Open(javaUSMAgentJarPath)
+	if err != nil {
+		log.Errorf("java TLS can't access to agent-usm.jar file %s : %s", javaUSMAgentJarPath, err)
+		return nil
+	}
+	jar.Close()
```

Comment:
Why Open? why not just `os.Stat` ?

Reply:
I did a stat but this not guarantee that we will able to copy this agent to container, so Open is more appropriate here